config/utils.go

```diff
@@ -87,28 +87,20 @@ func desanitize(actual, target *model.Config) {
 }
 
 // fixConfig patches invalid or missing data in the configuration, returning true if changed.
-func fixConfig(cfg *model.Config) bool {
-	changed := false
-
+func fixConfig(cfg *model.Config) {
 	// Ensure SiteURL has no trailing slash.
 	if strings.HasSuffix(*cfg.ServiceSettings.SiteURL, "/") {
 		*cfg.ServiceSettings.SiteURL = strings.TrimRight(*cfg.ServiceSettings.SiteURL, "/")
-		changed = true
 	}
 
 	// Ensure the directory for a local file store has a trailing slash.
 	if *cfg.FileSettings.DriverName == model.IMAGE_DRIVER_LOCAL {
 		if *cfg.FileSettings.Directory != "" && !strings.HasSuffix(*cfg.FileSettings.Directory, "/") {
 			*cfg.FileSettings.Directory += "/"
-			changed = true
 		}
 	}
 
-	if FixInvalidLocales(cfg) {
-		changed = true
-	}
-
-	return changed
+	_ = FixInvalidLocales(cfg)
```

Comment:
```suggestion
	FixInvalidLocales(cfg)
```

Why the empty assignment?

Reply:
explicitly ignoring return value